microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -366,6 +366,41 @@ If you find yourself running the same debug code in the **Console** tool over an
 See [Run snippets of JavaScript on any webpage](./snippets.md).
 
 
+<!-- ====================================================================== -->
+## Use `# sourceURL` to name evaluated and inline code in the Sources tool
+
+The `# sourceURL` pragma, with syntax `//# sourceURL=my-assigned-file-name.js` or `/*# sourceURL=my-assigned-file-name.css*/`, is a special comment that makes it easier to work with evaluated and inline code in the **Sources** tool.
+
+When a block of code is run by the browser, DevTools can only give that block of code a name if it was loaded from an network resource. When running the following kinds of code, the **Sources** tool doesn't have a file name to display these blocks of code in the **Navigator** pane:
```

Comment:
paragraph is only negative - negative is good but incomplete.  Specify positively and concretely in terms of UI, what will result in the UI if you do this.